docs/source/basics.rst

```diff
@@ -254,34 +254,3 @@ Below we show how we can define :math:`X_0\otimes \sqrt{X_1}` as a single gate.
 
     (0.5+0.5j)|01> + (0.5-0.5j)|11>
 
-Defining Parametric Gates
--------------------------
```

Comment:
I understand that we want to remove the `pyquil.parametric.ParametricProgram` references in the [Advanced Usage section](http://docs.rigetti.com/en/latest/advanced_usage.html#parametric-programs) since they use the same wording as the new Quil 2.0 feature, but I don't think we need to go so far as removing every feature that uses the word "parametric." Am I misunderstanding your reasoning for removing this section, @mpharrigan?

Reply:
my b. this shouldn't have been deleted